archive/progress_common/src/main/scala/uk/ac/wellcome/platform/archive/common/progress/monitor/ProgressTracker.scala

```diff
@@ -9,27 +9,29 @@ import com.gu.scanamo._
 import com.gu.scanamo.error.ConditionNotMet
 import com.gu.scanamo.syntax._
 import grizzled.slf4j.Logging
+import uk.ac.wellcome.platform.archive.common.models.BagId
 import uk.ac.wellcome.platform.archive.common.progress.models._
 import uk.ac.wellcome.storage.dynamo._
 
 import scala.concurrent.{ExecutionContext, Future, blocking}
 import scala.util.{Failure, Success, Try}
 
 class ProgressTracker @Inject()(
-  dynamoClient: AmazonDynamoDB,
-  dynamoConfig: DynamoConfig
+                                 dynamoDbClient: AmazonDynamoDB,
+                                 dynamoConfig: DynamoConfig
 )(implicit ec: ExecutionContext) extends Logging {
+
   import Progress._
 
   def get(id: UUID): Future[Option[Progress]] = Future {
-    blocking(Scanamo.get[Progress](dynamoClient)(dynamoConfig.table)(
+    blocking(Scanamo.get[Progress](dynamoDbClient)(dynamoConfig.table)(
```

Comment:
Why do we do this again? Could we not just pass a future by sequencing the futures here?